getting-started/organizations.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Orgs & sub-orgs
+description: How to manage your Edgee organization and sub-organizations.
+icon: list-tree
+---
+
+import EdgeeSdk from '/snippets/edgee-sdk.mdx';
+
+<EdgeeSdk />
+
+The Edgee platform allows you to organize your projects into **organizations**.
```

Comment:
that sounds weird

Reply:
i feel like this should be reworded, but i dont know how

Reply:
I will reword as 
> The Edgee platform allows you to create **Organizations**, then create projects and invite users into each organization.
> This is useful for managing multiple projects, especially if you are working with different teams or clients.
> You can create multiple organizations and invite different users to each organization.
This way, you can manage access to your projects more effectively.